apps/fabric/package.json

```diff
@@ -19,6 +19,7 @@
     "@react-native/eslint-config": "^0.72.2",
     "@react-native/metro-config": "^0.72.9",
     "@tsconfig/react-native": "^3.0.0",
+    "@types/react": "^18.2.15",
```

Comment:
You don't need this anymore. Having this and RN's internal types may cause uncertainty with the source of truth